api/cpp/esp-idf/slint/include/slint-esp.h

```diff
@@ -38,9 +38,9 @@ struct SlintPlatformConfiguration
     slint::PhysicalSize size;
     /// The handle to the display as previously initialized by `bsp_display_new` or
     /// `esp_lcd_panel_init`.
-    esp_lcd_panel_handle_t panel;
+    esp_lcd_panel_handle_t panel = nullptr;
     /// The touch screen handle, if the device is equipped with a touch screen.
```

Comment:
I think the documentation should specify that nullptr is valid for `touch` (while i think it might not be valid for `panel`

Should it be perhaps named `panel_handle` / `touch_handle`? What do you think?

Reply:
I agree, on both accounts. Will change. Thanks :)